File: src/main/java/priv/bajdcc/LALR1/interpret/os/task/TKUtil.java
package priv.bajdcc.LALR1.interpret.os.task;

import priv.bajdcc.LALR1.interpret.os.IOSCodePage;
import priv.bajdcc.util.ResourceLoader;

/**
 * 【服务】工具
 *
 * @author bajdcc
 */
public class TKUtil implements IOSCodePage {
	@Override
	public String getName() {
		return "/task/util";
	}

	@Override
	public String getCode() {
		return ResourceLoader.load(getClass());
	}
}

File: src/main/java/com/bajdcc/LALR1/interpret/os/task/TKUtil.kt
package com.bajdcc.LALR1.interpret.os.task

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【服务】工具
 *
 * @author bajdcc
 */
class TKUtil : IOSCodePage {
    override val name: String
        get() = "/task/util"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
